### DISCUSSION TITLE
Welcome to whatsapp-connector-rasa Discussions!

### DISCUSSION THREAD

**Discussion by maharanasarkar** [maharanasarkar/whatsapp-connector-rasa#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mumbaigirl** [comment]
@maharanasarkar **I have tried your code and it worked very well!**
Could you please tell me how can we send interactive messages like native whatsapp interactive menu , Here is the [link](https://developers.facebook.com/docs/whatsapp/guides/interactive-messages/), 
I want that when message comes from whatsapp webhook , Rasa should detect that this message is from a whatsapp user, rasa should adapt the  response accordingly, I mean right now , Rasa sends a plain text or can send a payload button, however, I want if the message arrives from whatsapp webhook , Rasa should adapt the replies accordingly.
![185251026_214443210216177_123641285902243447_n](https://github.com/maharanasarkar/whatsapp-connector-rasa/assets/99070985/bab0dfda-d7b2-4663-bf4a-ed7e81927aad)

**Reply by abhishek-bairagi** [reply]
hi @thatcuteseller , were you able to figure out how to send the List Message, I am able to send the button, but there is a limit of only 3 buttons max.  I would like to do something like the List Message image above

**Reply by maharanasarkar** [reply]
@abhishek-bairagi hey, the connetor uses an old version of heyoo in which the list support was not implemented. I am working on writing a WhatsApp cloud api wrapper in python and will update the connector based on the same package.
I will release  the wrapper as a package which will contain the latest api offerings from whatsapp cloud api

**Reply by abhishek-bairagi** [reply]
@maharanasarkar  Thank you! Looking forward to it.
